README.md

```diff
@@ -1,12 +1,23 @@
 # Microsoft Authentication Library for JavaScript (MSAL.js)
 
-The Microsoft Authentication Library for JavaScript enables client-side JavaScript web applications, running in a web browser, to authenticate users using [Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview) for work and school accounts (AAD), Microsoft personal accounts (MSA), and social identity providers like Facebook, Google, LinkedIn, Microsoft accounts, etc. through [Azure AD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-overview#identity-providers) service. It also enables your app to get tokens to access [Microsoft Cloud](https://www.microsoft.com/enterprise) services such as [Microsoft Graph](https://graph.microsoft.io).
+The Microsoft Authentication Library for JavaScript adds Auth to Javascript applications, and access to resource APIs in the [Microsoft Cloud](https://www.microsoft.com/enterprise) such as [Microsoft Graph](https://graph.microsoft.io).
 
-## Repository
+MSAL.js now supports Node.js and Electron! 
 
-### Core and wrapper libaries
+MSAL.js supports both consumer and enterprise Microsoft accounts.  This can be restricted if desired.  Only applications that use special consumer or enterprise data will need to differentiate.
 
-The [`lib`](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib) folder contains the source code for all of our libraries. You will also find all the details about **installing the libraries**, in their respective README.md.
+MSAL.js supports social identity providers like Facebook, Google, and LinkedIn through the [Azure AD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-overview#identity-providers) service. 
+
+## How do I get Started?
+
+Follow the link that matches your app:
+- [Traditional web app](lib/msal-browser/)
```

Comment:
Do we want to say "single-page applications"? To align with how it is labeled in teh portal. Traditional web app can mean a server-side app in our nomenclature.

Reply:
+1